lms/djangoapps/instructor/views/api.py

```diff
@@ -1047,6 +1057,7 @@ def re_validate_invoice(obj_invoice):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@sudo_required
 def get_students_features(request, course_id, csv=False):  # pylint: disable=redefined-outer-name
```

Comment:
As I wrote earlier, I'm afraid that decorating all these functions with `sudo_required` won't necessarily yield the results you expect.